node/src/actor_system_factory.rs

```diff
@@ -87,12 +85,10 @@ impl ActorSystemFactoryReal {
 pub trait ActorSystemFactoryTools {
     fn prepare_initial_messages(
         &self,
-        cryptdes: CryptDEPair,
```

Comment:
The `CryptDEPair` is now in `BootstrapperConfig`, so it doesn't have to be sent separately here.